test/fixtures/v8-coverage/sigint.js

```diff
@@ -5,3 +5,5 @@ if (true) {
   const c = 102;
 }
 process.kill(process.pid, "SIGINT");
+
+//# sourceMappingURL=https://http.cat/402
```

Comment:
Although this is just a fixture...I wonder whether we should support at least fetching the source maps from file system in core. We could delegate remote fetching to tooling, of course, but I imagine it would be nice if core supports some the most basic local fetching.

Reply:
I'm glad you feel this way :smile: 

I was trying to keep the PR as small as possible, but I actually like the idea of:

1. loading Base64 encoded data URLs if we observe them.
1. detecting file URLs and also loading these for folks.

Reply:
I've added support for loading source-maps from a file, the format of the `sourceMapCache` is:

```js
{
  url // raw sourceMapURL,
  data // sourceMap data (if we can load it) or null.
}
```

Reply:
I've added support for inline source-maps, and `.map` files.